### ISSUE TITLE
Have a catch-all for commands that don't exist.  Provide useful information on potential commands user CAN use

### ISSUE THREAD

**Issue by SpencerCDixon** [SpencerCDixon/redux-cli#10]
```
program
  .command('*')
  .action(function(env){
    console.log('Enter a Valid command');
    terminate(true);
});
```

Something like this

**Comment by jamesr73** [comment]
_Seemed_ simple enough:

```javascript
program
  .command('*')
  .action(function(cmd, env) {
    program.help()
  });
```

Except that this also intercepts the built-in `redux help <cmd>` such that the top level help is returned rather than the command specific help.

Which lead to this:

```javascript
program
  .command('*')
  .action(function(cmd, env) {
    if (cmd === 'help') {
      program.emit('--help')
    } else {
      program.help();
    }
  });
```

Which works if you ask it nicely:

`redux help generate`

But barfs on missing command if you're lazy:

`redux help`

And barfs by running the command (?!) if you're over zealous:

`redux help generate duck`

Are you interested in a PR if I continue trying to refine this?

**Comment by SpencerCDixon** [comment]
Yeah if you get it working I'll definitely merge.  Was probably going to start putting some more time into this project now that I'm settled into my new job/not ridiculously busy haha.

Appreciate you wanting to contribute!!

**Comment by jamesr73** [comment]
Cool. I'll see if I can do any better. I used yargs on my last project and it has similar quirks ;-)

**Comment by anithri** [comment]
Addressed by #110 implemented `yargs` and has been merged.
